### PR TITLE
feat(cohorts): Add a text index on cohort slugs

### PR DESCRIPTION
**Pull request by arku**
This is required to implement search functionality on cohorts.

### CODE REVIEW

File: src/schemas/CohortSchema.js [hunk 1]
```diff
@@ -38,5 +38,7 @@ module.exports = (conn) => {
     },
   });
 
+  CohortSchema.index({ slug: 'text' });
```
Comment (raulingg) [inline src/schemas/CohortSchema.js:41]: Why text?

**Comment by arku** [comment]
> I wonder if we should add an index for name key, because many will search by name.

@raulingg Doesn't slug include the name?

**Comment by arku** [comment]
> I'm not sure about that

@raulingg It does. The last part of the slug is the name.

**Comment by raulingg** [comment]
> > I'm not sure about that
> 
> @raulingg It does. The last part of the slug is the name.

Cool!

**Pull request merged**